crates/sui-graphql-rpc/src/types/zklogin_verify_signature.rs

```diff
@@ -130,7 +133,13 @@ pub(crate) async fn verify_zklogin_signature(
                 return Err(Error::Client("Tx sender mismatch author".to_string()));
             }
             let sig = GenericSignature::ZkLoginAuthenticator(zklogin_sig);
-            match sig.verify_authenticator(&intent_msg, tx_sender, curr_epoch, &verify_params) {
+            match sig.verify_authenticator(
+                &intent_msg,
+                tx_sender,
+                curr_epoch,
+                &verify_params,
+                Arc::new(VerifiedDigestCache::new_for_testing()),
```

Comment:
so for graphql purposes we'll always pass in an empty cache, aka it wouldn't be any diff from the current implementation?

Reply:
yes